src/components/pages/Resources.js

```diff
@@ -111,6 +112,7 @@ class Resources extends Component {
     };
 
     componentDidMount() {
+        analytics.logEvent("resources_page_view");
```

Comment:
Should this not be on every page?

Reply:
I've added this to the profile page. It will be added to the others as we build the pages (not sure on the structure as of now).

Reply:
What about the index page?

Reply:
Firebase automatically does that one. 🤷‍♂️

Reply:
Ah perfect